packages/messaging/lib/sendMessageWrapper.js

```diff
@@ -0,0 +1,144 @@
+import { sendMessage } from '../../../src/utils'
+
+export class SendMessageMessagingConfig {}
+
+/**
+ * An temporary implementation of {@link MessagingTransport} to communicate with Android and Extension.
```

Comment:
If this is temporary, I suggest marking it as such by class name.

I think you can mark it as `@deprecated` immediately too.

Reply:
On a call with @franfaccin - I showed how to achieve this without modifying the underlying Messaging framework 💪🏻 

see: https://github.com/duckduckgo/content-scope-scripts/compare/shane/pr-review?expand=1#diff-bad7f5b7c7b37884f33af05ead612b430d72337fa089568d10e0cb7127e84516

The short version is that we now make it clear this is a CTL extension to messaging, and we handle stateful things with composition instead of ad-hoc changes to the messaging library :)